sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/home/Item.kt

```diff
@@ -1,29 +1,48 @@
 package co.nimblehq.sample.compose.ui.screens.home
 
-import androidx.compose.foundation.clickable
+import androidx.compose.foundation.*
 import androidx.compose.foundation.layout.*
-import androidx.compose.material.Text
-import androidx.compose.runtime.Composable
+import androidx.compose.material.*
+import androidx.compose.runtime.*
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.tooling.preview.Preview
+import co.nimblehq.sample.compose.R
 import co.nimblehq.sample.compose.model.UiModel
+import co.nimblehq.sample.compose.ui.theme.*
 import co.nimblehq.sample.compose.ui.theme.AppTheme.dimensions
-import co.nimblehq.sample.compose.ui.theme.ComposeTheme
 
+@OptIn(ExperimentalFoundationApi::class)
 @Composable
 fun Item(
     uiModel: UiModel,
-    onClick: (UiModel) -> Unit
+    onClick: (UiModel) -> Unit,
+    onLongClick: (UiModel) -> Unit
 ) {
-    Row(
+    var expanded by remember { mutableStateOf(false) }
+
+    Box(
         modifier = Modifier
             .fillMaxWidth()
-            .clickable(onClick = { onClick(uiModel) })
+            .combinedClickable(
+                onClick = { onClick(uiModel) },
+                onLongClick = { expanded = !expanded }
```

Comment:
Optional, might be easier to read

```suggestion
                onLongClick = { expanded = true }
```